lib/vc.js

```diff
@@ -162,7 +310,92 @@ function _getId(obj) {
   return obj.id;
 }
 
-async function _checkCredential({credential}) {
+/**
+ * Checks if a given credential is a VerifiablePresentation.
+ *
+ * @param credential
+ * @returns {boolean}
+ */
+function _isPresentation(credential) {
+  return credential['type'] &&
+    jsonld.getValues(credential, 'type').includes('VerifiablePresentation');
+}
+
+/**
+ * @param presentation
+ * @throws {Error}
+ * @private
+ */
+function _checkPresentation(presentation) {
+  if(!presentation['@context']) {
+    throw new Error('"context" property is required.')
```

Comment:
All these cred/pres errors should have the `@` to avoid confusion: `"@context"`.

Reply:
Changed.